roles/variables/vars/main.yml

```diff
@@ -606,6 +606,11 @@ kafka_broker_properties:
     enabled: "{{ rbac_enabled and not external_mds_enabled and sso_mode != 'none' and sso_groups_scope != 'none' }}"
     properties:
       confluent.oidc.idp.groups.claim.scope: "{{ sso_groups_scope }}"
+  rbac_mds_sso_ssl:
+    enabled: "{{ rbac_enabled and (not external_mds_enabled) and sso_mode != 'none' and sso_idp_cert_path != ''}}"
```

Comment:
why do we not truststore in case of external mds

Reply:
we only need to add trusstore on mds server, in case of ext mds kafka acts as mds client